src/components/StakingPage/StakePositionTable/index.tsx

```diff
@@ -91,48 +93,53 @@ export default memo(function StakePositionTable(props: IProps) {
           <Switch id="stake-position-switch" className="switch" label="Show archive" checked={showArchive} onChange={handleArchive}></Switch>
         </Box>
       </Flex>
-      <div sx={{
-        mb: '100px',
-      }}>
-        {tableData.length > 0 && (
-          <Table rowKey="index" sx={styles.tableWrapper} data={tableData}>
-            <Column<ITableData>
-              title=""
-              dataIndex="icon"
-              key="icon"
-              render={renderVaultIcon}
-              width={72}
-            ></Column>
+      {
+        tableData.length ? (isMobile &&

```

Comment:
I think this condition is too complicated.

Reply:
restructure with lodash._cond